tests/foreman/maintain/test_service.py

```diff
@@ -215,11 +217,11 @@ def test_positive_status_clocale(sat_maintain):
     :parametrized: yes
 
     :steps:
-        1. Run LC_ALL=C satellite-maintain service stop
+        1. Run LC_ALL=C.UTF-8 satellite-maintain service status
 
     :expectedresults: service status works with C locale
     """
-    assert sat_maintain.cli.Service.status(env_var='LC_ALL=C').status == 0
+    assert sat_maintain.cli.Service.status(env_var='LC_ALL=C.UTF-8').status == 0
```

Comment:
We could test here that the result is not gibberish due to encoding.

Reply:
or how about using lang locales like "en_US.UTF-8" here to avoid gibberish results?